ansible/playbook/README.md

```diff
@@ -45,12 +52,27 @@ Ansible works against multiple systems in an infrastructure, called “hosts”,
 Once the inventory is defined, you use patterns to select the hosts or groups you want Ansible to run against.
 
 The inventory is comprised of several files and scripts based at the `k8s-infra/ansible/inventory` folder.
+* `hosts.yml`: YAML file with group structure information as well as group variable assignment
+* `pass_inventory.py`: Python script that dynamically builds the inventory from the `passwordstore` project for use in the Ansible Playbooks. This task is done automatically when Ansible is executed. 
 
-The inventory is built from the `passwordstore` project using the `pass_inventory.py` python script. This task is done automatically when Ansible is executed. 
 For more information on the `passwordstore` project check gitlab.
 
 More information on the Ansible Inventory and how to build it in the [Ansible User Guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).
 
+# `hosts.yml`
+
```

Comment:
Can you add an example about the `hosts.yml` to be created as most of us have no idea about what it should contain ? @jacobdotcosta